include/ncengine/graphics/ToonRenderer.h

```diff
@@ -15,15 +15,15 @@ namespace nc::graphics
 struct ToonMaterial
 {
     std::string baseColor = asset::DefaultBaseColor;
-    std::string overlay = asset::DefaultBaseColor;
+    uint32_t outlineWidth = 2;
```

Comment:
This is a breaking change for serialization. We have a version number in the scene fragment, but, at least for now, no backwards compatible loading. Are we ok never updating SS&DD further? If so, this can stay, but the version in `SceneSerialization.h` needs to be bumped.